ops-tools/docker/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.8"
+
+services:
+  Postgres:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: school
+      POSTGRES_DB: school
+    ports:
+      - 5432:5432
+    volumes:
+      - ./data/postgres:/var/lib/postgresql/data
+      - ./data/dump:/db-dump
+    networks:
+      - docker-practice-net
+
+  Redis:
+    image: redis:latest
```

Comment:
When we use `latest` tag for docker images - we can face with issues when the version of that image was changed and our code will not work with that versions of docker images. So, again, please set version of docker image explicitly. Thanks